pragma-node/src/handlers/entries/mod.rs

```diff
@@ -29,10 +29,29 @@ use crate::{
     utils::doc_examples,
 };
 
+pub trait SignedRequest {
```

Comment:
I'm not sure of the utility of this trait. Couldn't we just implement AsRef ?

Reply:
yeah true - a little less clear for me but that works, done + tested